src/components/ProductList/ProductList.stories.tsx

```diff
@@ -1,6 +1,114 @@
 import type { Meta, StoryObj } from '@storybook/react';
 import ProductList from './ProductList';
+import { Product } from '../../types/product';
+import images from '../../assets/images/jacket.jpg';
 
+const Products: Product[] = [
+  {
+    title: 'title 1',
+    price: 14,
+    images: [images],
+    colors: [],
+    sizes: ['blue', 'whilte'],
+    description: 'description 1',
+    stock: 45,
+    rate: 31,
+    numberRating: 2,
+    category: '1',
+    id: 1,
+  },
+  {
+    title: 'title 2',
+    price: 61,
+    images: [images],
+    colors: [],
+    sizes: ['blue', 'whilte'],
+    description: 'description 2',
+    stock: 83,
+    rate: 99,
+    numberRating: 4,
+    category: '2',
+    id: 2,
+  },
+  {
+    title: 'title 3',
+    price: 40,
+    images: [images],
+    colors: [],
+    sizes: [],
+    description: 'description 3',
+    stock: 6,
+    rate: 73,
+    numberRating: 3,
+    category: 'category 3',
```

Comment:
Use real name of category, like `T-shirt`

Reply:
Fixed